src/linux/Packaging.Linux/pack-tool.sh

```diff
@@ -0,0 +1,70 @@
+#####################################################################
+# Pack dotnet tool
+#####################################################################
+echo "Creating dotnet tool package..."
+# Parse script arguments
+for i in "$@"
+do
+case "$i" in
+    --payload=*)
+    PAYLOAD="${i#*=}"
+    shift # past argument=value
+    ;;
+    --version=*)
+    VERSION="${i#*=}"
+    shift # past argument=value
+    ;;
+    --release=*)
+    RELEASE="${i#*=}"
+    shift # past argument=value
+    ;;
```

Comment:
What is this argument used for? I don't see a place where we call the script with this.

Reply:
It defaults to the location we expect when running in GH Actions (definitely the happy path), but if you run it locally you need to specify this directory ([here's an example](https://github.com/GitCredentialManager/git-credential-manager/pull/886#issuecomment-1267698180) from my local install instructions).

Reply:
Ended up removing after all!